MyCryptoHQ/MyCrypto
# Refactor browser compatibility check

## Description

I refactored the browser checks to use Modernizr instead of our own implementation.

## Changes

* Removed old browser checks
* Implemented `webpack-modernizr-loader` for small Modernizr builds
* Added new browser check using Modernizr

## Steps to Test

1. Disable one of the required features (e.g. local storage)
2. Open the website
3. Check if the bad browser overlay shows up

## Quality Assurance
